.gitignore

```diff
@@ -31,3 +31,5 @@ build/
 
 ### VS Code ###
 .vscode/
+
+.env
```

Comment:
Remove it from the remote repo using: 
`git rm --cached .env` - .env should never be in version control